control/narrative_table.go

```diff
@@ -0,0 +1,91 @@
+package control
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/jbowtie/gokogiri/xml"
+	"github.com/opencontrol/fedramp-templater/docx/helper"
+	"github.com/opencontrol/fedramp-templater/opencontrols"
+)
+
+func findSectionKey(row xml.Node) (section string, err error) {
+	re := regexp.MustCompile(`Part ([a-z])`)
```

Comment:
What I pasted above is an extra table.
The original table for AC-8 is (works as expected):
![image](https://cloud.githubusercontent.com/assets/7788930/18033535/d39343f6-6cf5-11e6-9e17-7cf5c78ed545.png)

Then there's extra info like this:
![image](https://cloud.githubusercontent.com/assets/7788930/18033536/e4860af4-6cf5-11e6-8cfa-986a64f779d2.png)
 Then there's the table I pasted in the first comment.
This is something that the yaml doesn't take care of yet. So tl;dr: false alarm.